scripts/version.js

```diff
@@ -0,0 +1,154 @@
+
+        // configuration-api
+        process.stdout.write("Moving configuration-api...");
+
+        shell.mkdir(`./content/management/configuration-api/v${fromVersion}`);
+        shell.exec(
+          `git mv ./content/management/configuration-api/index.mdx ./content/management/configuration-api/v${fromVersion}`
+        );
+        shell.exec(
+          `git mv ./content/management/configuration-api/v${toVersion}/index.mdx ./content/management/configuration-api/`
+        );
+
+        process.stdout.write("Done\n");
+
+        // agent-chat-api
+        process.stdout.write("Moving agent-chat-api...");
+        shell.mkdir(`./content/messaging/agent-chat-api/v${fromVersion}`);
+        shell.mkdir(
+          `./content/messaging/agent-chat-api/v${fromVersion}/rtm-reference`
+        );
+        shell.exec(
+          `git mv ./content/messaging/agent-chat-api/index.mdx ./content/messaging/agent-chat-api/v${fromVersion}`
+        );
+        shell.exec(
+          `git mv ./content/messaging/agent-chat-api/v${toVersion}/index.mdx ./content/messaging/agent-chat-api/`
+        );
+        shell.exec(
+          `git mv ./content/messaging/agent-chat-api/rtm-reference/index.mdx ./content/messaging/agent-chat-api/v${fromVersion}/rtm-reference`
+        );
+        shell.exec(
+          `git mv ./content/messaging/agent-chat-api/v${toVersion}/rtm-reference/index.mdx ./content/messaging/agent-chat-api/rtm-reference`
+        );
+        shell.rm("-R", `./content/messaging/agent-chat-api/v${toVersion}`);
+        process.stdout.write("Done\n");
+
+        // customer-chat-api
+        process.stdout.write("Moving customer-chat-api...");
+        shell.mkdir(`./content/messaging/customer-chat-api/v${fromVersion}`);
+        shell.mkdir(
+          `./content/messaging/customer-chat-api/v${fromVersion}/rtm-reference`
+        );
+        shell.exec(
+          `git mv ./content/messaging/customer-chat-api/index.mdx ./content/messaging/customer-chat-api/v${fromVersion}`
+        );
+        shell.exec(
+          `git mv ./content/messaging/customer-chat-api/v${toVersion}/index.mdx ./content/messaging/customer-chat-api/`
+        );
+        shell.exec(
+          `git mv ./content/messaging/customer-chat-api/rtm-reference/index.mdx ./content/messaging/customer-chat-api/v${fromVersion}/rtm-reference`
+        );
+        shell.exec(
+          `git mv ./content/messaging/customer-chat-api/v${toVersion}/rtm-reference/index.mdx ./content/messaging/customer-chat-api/rtm-reference`
+        );
+        shell.rm("-R", `./content/messaging/customer-chat-api/v${toVersion}`);
+        process.stdout.write("Done\n");
+
+        console.log(
+          "Remember to update STABLE_VERSION and LEGACY_VERSIONS in constants"
+        );
+
+        shell.config.silent = silentState;
+      } else {
+        console.log("--fromVersion and --toVersion are required");
+        shell.exit(1);
+      }
+      break;
+    }
+
+    case "new": {
```

Comment:
We could add two messages for this case:
 "Remember to update apiVersion in the document's meta" 
"Remember to update ALL_VERSIONS in constants"